tests/phpunit/includes/options-trait.php

```diff
@@ -14,9 +14,14 @@ trait PLL_Options_Trait {
 	 * @since 3.7
```

Comment:
The tag is now meaningless IMHO.

Reply:
Don't you dare touch my `@since` tags! 🔪